### PR TITLE
Common tests for EntityRepository

### PR DESCRIPTION
**Pull request by dmdashenkov**
Common tests for `EntityRepository`:
 - `load(id)`
 - `loadAll(ids)`
 - `find(EntityFilters, FieldMask)`

Fixed issue with returning multiple `null` values from `InMemoryEntityRepository`.

### CODE REVIEW

**Comment by dmdashenkov** [comment]
@armiol PTAL

File: server/src/test/java/org/spine3/server/entity/AbstractEntityRepositoryShould.java [hunk 1]
```diff
@@ -105,6 +112,58 @@ public void handle_wrong_passed_ids() {
...
+
+        final int count = 10;
+        final List<E> entities = entities(count);
+
+        for (E entity : entities) {
+            repo.store(entity);
+        }
+
+        final List<EntityId> ids = new LinkedList<>();
+        for (int i = 0; i < count / 2; i++) {
```
Comment (armiol) [inline server/src/test/java/org/spine3/server/entity/AbstractEntityRepositoryShould.java]: Please comment this piece of code. It's not absolutely clear, what you want to achieve.

File: server/src/test/java/org/spine3/server/entity/AbstractEntityRepositoryShould.java [hunk 2]
```diff
@@ -105,6 +112,58 @@ public void handle_wrong_passed_ids() {
...
+
+        assertSize(ids.size(), readEntities);
+
+        for (E entity : readEntities) {
+            assertMatches(entity, firstFieldOnly);
+        }
+    }
+
+
+    private void assertMatches(E entity, FieldMask fieldMask) {
```
Comment (armiol) [inline server/src/test/java/org/spine3/server/entity/AbstractEntityRepositoryShould.java]: `static`?

Comment (dmdashenkov) [inline server/src/test/java/org/spine3/server/entity/AbstractEntityRepositoryShould.java]: Done

**Comment by dmdashenkov** [comment]
@armiol PTAL

**Pull request merged**